test/integration/integration_suite_test.go

```diff
@@ -191,8 +214,22 @@ func getRegionsWithCaps(t *testing.T, client *linodego.Client, capabilities []st
 		return true
 	}
 
+	// Function to check if a region has the required plans available
+	regionHasPlans := func(regionID string) bool {
+		if len(plans) == 0 {
```

Comment:
Still left this check in to avoid any unnecessary map lookups when there are no plans to check.